README.md

```diff
@@ -46,7 +46,8 @@ Run `build.cmd /t:Test` or `build.sh /t:Test`
 ## Run end-to-end tests
 
 Prerequisites:
-- Install [selenium-standalone](https://www.npmjs.com/package/selenium-standalone) (requires Java 8 or later)
+- Install [selenium-standalone](https://www.npmjs.com/package/selenium-standalone) (requires Java 8 or 9)
```

Comment:
Java 10 is out. Does it not work with Selenium?

Reply:
Nope, hence my grief yesterday after I figured out after 2h